docs/src/modules/components/AppSearch.js

```diff
@@ -19,11 +19,20 @@ import useLazyCSS from 'docs/src/modules/utils/useLazyCSS';
 
 const SearchButton = styled('button')(({ theme }) => {
   return {
-    display: 'none',
     minHeight: 33,
+    display: 'flex',
+    alignItems: 'center',
+    paddingLeft: theme.spacing(1),
+    [theme.breakpoints.only('xs')]: {
+      backgroundColor: 'transparent',
+      padding: 0,
+      minWidth: 33,
+      justifyContent: 'center',
+      '& > *:not(:first-child)': {
```

Comment:
@siriwatknp This creates a warning:

<img width="553" alt="Screenshot 2021-10-31 at 01 52 17" src="https://user-images.githubusercontent.com/3165635/139561443-e9167f9d-b17a-4396-a3dd-0ed6b99b7192.png">